src/components/date-picker.js

```diff
@@ -1085,15 +1085,17 @@ const DatePicker = ({
               </Menu.Button>
               :
               <>
-                {label.title && <div className={labelClasses.container}>
-                  <p className={labelClasses.title}>{label.title}</p>
+                { (label.title || typeof label === 'string') && <div className={labelClasses.container}>
+                  <p className={labelClasses.title}>
+                    {typeof label === 'string' ? label : label.title}
+                  </p>
```

Comment:
does `<p className={labelClasses.title}>{label?.title || label}</p>` give an error?
if not would prefer this to simplify it a little

Reply:
yea it works